src/schema/v2/artwork/index.ts

```diff
@@ -913,6 +913,12 @@ export const ArtworkType = new GraphQLObjectType<any, ResolverContext>({
           return comparables_count > 0 && category !== "Architecture"
         },
       },
+      isEcommerce: {
```

Comment:
Its a bit odd to me that this has never been added to our schema before, but maybe its mostly just useful in CMS.

Reply:
I think this is a raw field. More like 'galley signals intention' if it wants the work to be `buy now`. The `isAcquireable` is the one that clients use to actually see if `buy now` transaction is possible.